src/usbs/usbs.h

```diff
@@ -1,9 +1,11 @@
+#include <systemd/sd-device.h>
```

Comment:
This `include` here disturb my mind. :scream: 
It's here to in order to compile/run the tests...